### PR TITLE
Added R-Shiny to the Front-End Development secion.

### PR DESCRIPTION
**Pull request by grabear**
**https://github.com/grabear/awesome-rshiny**

**This awesome list is a compilation of useful R-Shiny resources.  R-shiny has been available for several years now, and has evolved into a highly useful package for Data Scientists who are not necessarily familiar with web development.  It gives these scientist an opportunity to create great web application for their publications and affiliated institutions.  Accepting this link into the Front End Development section will bring more awareness to the inherit usefulness of Shiny as a web framework.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by grabear** [comment]
https://github.com/grabear/awesome-rshiny/issues/9

**Comment by sindresorhus** [comment]
Please read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) more closely. Multiple points not followed from it. Try going through the list point for point and check that you follow it. :)

**Comment by ucg8j** [comment]
Hi @sindresorhus a number of changes have been made over the last couple of weeks, many to address the awesome guidelines. Would you be able to take another look at this PR and advise on any points we're still not meeting?

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

File: readme.md [hunk 1]
```diff
@@ -230,9 +230,11 @@
 - [Preact](https://github.com/ooade/awesome-preact) - App framework.
 - [Progressive Enhancement](https://github.com/jbmoelker/progressive-enhancement-resources)
 - [Next.js](https://github.com/unicodeveloper/awesome-nextjs) - Framework for server-rendered React apps.
+- [R-Shiny](https://github.com/grabear/awesome-rshiny) - Web Framework built with R.
 - [Hyperapp](https://github.com/hyperapp/awesome-hyperapp) - Tiny JavaScript library for building web apps.
 
 
+
```
Comment (DanailMinchev) [inline readme.md]: This line is not needed, please remove it

File: readme.md [hunk 2]
```diff
@@ -230,9 +230,11 @@
 - [Preact](https://github.com/ooade/awesome-preact) - App framework.
 - [Progressive Enhancement](https://github.com/jbmoelker/progressive-enhancement-resources)
 - [Next.js](https://github.com/unicodeveloper/awesome-nextjs) - Framework for server-rendered React apps.
+- [R-Shiny](https://github.com/grabear/awesome-rshiny) - Web Framework built with R.
```
Comment (DanailMinchev) [inline readme.md]: As per rules in https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md#requirements-for-your-pull-request you should add the entry in the end of the category, so you need to add it after `Hyperapp`

Comment (DanailMinchev) [reply readme.md]: also, you should add `#readme` in the end of the link and make sure it is linked proprly

Comment (DanailMinchev) [reply readme.md]: No need to specify the section name in your PR title, please rename it from `Added R-Shiny to the Front-End Development secion.` to `Add R-Shiny`

Comment (DanailMinchev) [reply readme.md]: please also rebase your PR as it has conflicts currently

**Comment by sindresorhus** [comment]
@grabear :arrow_up:

**Comment by grabear** [comment]
@sindresorus I will try to lint this myself tomorrow and make some changes.  Been busy!  :)

**Comment by sindresorhus** [comment]
Closing for lack of activity. The repo also has not been updated since mid last year.

**Pull request closed without merging**